internal/plugin/connectors/http/template/connector.go

```diff
@@ -0,0 +1,24 @@
+package template
```

Comment:
This location for the template connector isn't optimal. Generally we don't want any unused code in `internal/`